app/models/concerns/fe/choice_field_concern.rb

```diff
@@ -15,7 +15,7 @@ module ChoiceFieldConcern
       included do
         has_many :elements, class_name: "Element", foreign_key: "choice_field_id", dependent: :nullify#, order: :position
         [:rating_before_label_translations, :rating_after_label_translations, :rating_na_label_translations].each do |column|
-          if Rails::VERSION::MAJOR < 7
+          if Rails.gem_version < "7.1.0"
```

Comment:
Interesting, I didn't know of that way to compare rails versions. Turns out if you have the LHS be Gem::Version it'll convert your string to a Gem::Version also and it's not just doing string comparison.